utils/src/main/scala/cromwell/pipeline/utils/ApplicationConfig.scala

```diff
@@ -3,8 +3,64 @@ package cromwell.pipeline.utils
 import com.typesafe.config.{ Config, ConfigFactory }
 import pdi.jwt.JwtAlgorithm
 import pdi.jwt.algorithms.JwtHmacAlgorithm
+import play.api.libs.json.{ JsObject, Json, Writes }
 
 sealed trait ConfigComponent
+object ConfigComponent {
+  implicit val webserviceWrites = new Writes[WebServiceConfig] {
+    def writes(config: WebServiceConfig) = Json.obj(
+      "interface" -> config.interface,
+      "port" -> config.port
+    )
```

Comment:
Use typesafe json combinators instead. (https://www.playframework.com/documentation/2.8.x/ScalaJsonCombinators#Writes).
If someone will add a new field to a config, your implementation of `Writes` will compile. Sometimes it really is what you need, but in this case it is not. Using a combinator will ensure that if someone adds a field to the config, the code will not compile until he fixes `Writer`.